addons/mail/static/src/owl/components/composer/composer.js

```diff
@@ -115,7 +115,7 @@ class Composer extends Component {
      * @return {boolean}
      */
     get hasFooter() {
-        return this.storeProps.composer.attachmentLocalIds.length > 0;
+        return this.storeProps.composer.attachmentLocalIds.length > 0 || !this.props.areButtonsInline;
```

Comment:
This looks wrong: discuss app and chat windows have composer with inline buttons, and it has footer when there are some attachments

Reply:
It's a or, so if the buttons are not inline or if there are attachments then the footer is printed.
I don't see the problem here ? Maybe you mean that the footer part should only be used for attachments ?

Reply:
Oops my bad, thought it was `&&`.
Not sure it should always display a footer in non-inline buttons mode, but you can keep that way for the moment.

Reply:
Okay :) !